objects/object_ptr.go

```diff
@@ -39,3 +39,13 @@ func (o *ObjectPtr) IsFalsy() bool {
 func (o *ObjectPtr) Equals(x Object) bool {
 	return o == x
 }
+
+// IndexGet returns an element at a given index.
```

Comment:
Should an `ObjectPtr` be indexable / index-settable if the underlying object is?

Reply:
Tengo doesn't have pointers. `ObjectPtr` is used only internally and should never require indexing.

Reply:
#194 fixes #193 so this should be resolved now.